### PR TITLE
Feature/tandem filters colors

### PR DESCRIPTION
**Pull request by mjoyce91**
~~Relies on #882 (merge first)~~

Screen Shot 2020-05-05 at 3 49 25 PM

-------------------------------------------------------------------

Screen Shot 2020-05-05 at 3 49 31 PM

### CODE REVIEW

**Comment by elizabeth-jimenez** [comment]
image
To replicate: Click Tandem2 and then toggle back to non-Tandem search, then toggle back to Tandem search. Current User will have Tandem2's filter colors.

File: src/Components/SearchFilters/SearchFiltersContainer/SearchFiltersContainer.jsx [hunk 1]
```diff
@@ -370,38 +374,49 @@ class SearchFiltersContainer extends Component {
...
+              <TandemSelectionFilter onChange={this.onTandemSelectionClick} />
+            </div>
           }
           {
             tandemIsSelected && !showTandem2 &&
-            <MultiSelectFilterContainer
-              multiSelectFilterList={sortedFiltersTandem1}
-              queryParamToggle={this.props.queryParamToggle}
-            />
+            <div className={tandem1Class}>
```
Comment (elizabeth-jimenez) [inline src/Components/SearchFilters/SearchFiltersContainer/SearchFiltersContainer.jsx:405]: @mjoyce91 This placeholder class never seems to be applied. Toggling between Tandem1 and Tandem2 seems to just remove the `tandem2Class` class?

Comment (mjoyce91) [reply src/Components/SearchFilters/SearchFiltersContainer/SearchFiltersContainer.jsx:405]: Fixing now

Comment (mjoyce91) [reply src/Components/SearchFilters/SearchFiltersContainer/SearchFiltersContainer.jsx:406]: Updated

File: src/Components/SearchFilters/SearchFiltersContainer/SearchFiltersContainer.jsx [hunk 2]
```diff
@@ -361,6 +362,10 @@ class SearchFiltersContainer extends Component {
     });
 
     const apContainerClass = 'ap-container';
+    const commonContainerClass = tandemIsSelected ? 'tandem-common-filters' : '';
+    const tandem1Class = 'tandem-1-filters';
+    const tandem2Class = 'tandem-2-filters';
+    const tandemUserClass = showTandem2 ? 'tandem-2-filters' : '';
```
Comment (elizabeth-jimenez) [inline src/Components/SearchFilters/SearchFiltersContainer/SearchFiltersContainer.jsx:368]: @mjoyce91 Maybe we want to add `tandem-1-filters` placeholder class to the third ternary operator arg? It doesn't matter for this PR, so i'll go ahead and approve 👍

**Pull request merged**